src/main/kotlin/net/ccbluex/liquidbounce/config/types/Value.kt

```diff
@@ -362,6 +363,76 @@ class BindValue(
     }
 }
 
+class MultiChooseListValue<T>(
+    name: String,
+    value: Array<T>,
+
+    @Exclude
+    val choices: Array<T>,
+
+    @Exclude
+    @ProtocolExclude
+    private val clazz: KClass<T>
+) : Value<Array<T>>(
```

Comment:
Why not implement this with an [`EnumSet`](https://docs.oracle.com/javase/8/docs/api/java/util/EnumSet.html)?

You do an extra mile by converting from `Array<T>` -> `Set<T>` -> `Array<T>`